forms/static/forms/admin/dependency_fields.js

```diff
@@ -9,17 +9,40 @@
             $('.field-victim_of').hide();
             $('.field-survivor_of').hide();
         }
+
     });
     $(document).change(function() {
+        const removeValue = function(id) {
+            $(`#${id}`).children('option:first').removeAttr('value');
+        };
+        const addValue = function(id) {
+            $(`#${id}`).children('option:first').attr('value', '');
+        };
         let newspaper_person = $('#id_newspaperperson_set-0-victim_or_survivor_1:checked').length;
         let radio_person = $('#id_radioperson_set-0-victim_or_survivor_1:checked').length;
         let tv_person = $('#id_televisionperson_set-0-victim_or_survivor_1:checked').length;
         let internent_person = $('#id_internetnewsperson_set-0-victim_or_survivor_1:checked').length;
 
         if((newspaper_person > 0) || (radio_person > 0) || (tv_person > 0) || (internent_person > 0)){
+            addValue('id_newspaperperson_set-0-victim_of');
+            addValue('id_newspaperperson_set-0-survivor_of');
+            addValue('id_radioperson_set-0-victim_of');
+            addValue('id_radioperson_set-0-survivor_of');
+            addValue('id_televisionperson_set-0-victim_of');
+            addValue('id_televisionperson_set-0-survivor_of');
+            addValue('id_internetnewsperson_set-0-victim_of');
+            addValue('id_internetnewsperson_set-0-survivor_of');
```

Comment:
In a single sheet we can only have one of `newspaper_person`, `radio_person`, etc., right? So why not just handle that e.g.
```js
if((newspaper_person > 0) || (radio_person > 0) || (tv_person > 0) || (internent_person > 0)){
  if (newspaper_person > 0) {
     addValue('id_newspaperperson_set-0-victim_of');
     addValue('id_newspaperperson_set-0-survivor_of');
  }

  ....

```

Reply:
Oh... The `if` statement was already existing to toggle the fields, I just added my code to it. @kilemensi